Freshli.Web/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@model AnalysisRequest
+@{
+    ViewData["Title"] = "Home Page";
+}
+
+<div class="text-center">
+    <h1 class="display-4">Welcome</h1>
+    <p>Learn about <a href="https://docs.microsoft.com/aspnet/core">building Web apps with ASP.NET Core</a>.</p>
+</div>
+
+<div>
+    <form asp-route="CreateAnalysisRequest" method="post">
+        <div>
+            <label asp-for="Url">Public Git URL:</label>
+            <input asp-for="Url" type="url"/>
+        </div>
```

Comment:
This is nitpicking, but, unless the other two fields are optional, I'd move this after them.

Reply:
Created issue #60 to track getting this cleaned up and made more clear.